tools/android_sdk/create_cipd_packages.sh

```diff
@@ -116,12 +119,20 @@ for platform in "${platforms[@]}"; do
   done
 
   # Special treatment for NDK to move to expected directory.
+  # This is because the engine imports darts GN files which also inlcude
+  # a copy of the ndk. See https://github.com/flutter/flutter/issues/136666#issuecomment-1805467578
+  # for more information.
   mv $upload_dir/sdk/ndk $upload_dir/ndk-bundle
   ndk_sub_paths=`find $upload_dir/ndk-bundle -maxdepth 1 -type d`
   ndk_sub_paths_arr=($ndk_sub_paths)
   mv ${ndk_sub_paths_arr[1]} $upload_dir/ndk
   rm -rf $upload_dir/ndk-bundle
 
+  if [[ ! -d "$upload_dir/ndk" ]]; then
+  echo "Failure to bundle ndk for platform"
+  exit 1
```

Comment:
shouldn't these lines be indented?

btw I wonder if running [shellcheck](https://www.shellcheck.net) would make sense in Flutter's repos.

Reply:
Done, if you wanted to add shellcheck I am not opposed. I will say that we (the broader fluter team) have agreed to move from shell scripts like this to dart scripts. Thanks for the catch, this is fixed.